spinalcordtoolbox/resample/nipy_resample.py

```diff
@@ -0,0 +1,176 @@
+#########################################################################################
+#
+# Resample data.
+#
+# ---------------------------------------------------------------------------------------
+# Copyright (c) 2014 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Sara Dupont
+# Modified: 2017-11-22 (moved the code to spinalcordtoolbox module)
+#
+#
+# About the license: see the file LICENSE.TXT
+#########################################################################################
+import sys
+import commands
+from copy import deepcopy
+
+import sct_utils as sct
```

Comment:
You could rework the order of imports to [batteries included, 3rd party, internal] while you're at it.

Reply:
Done